### PR TITLE
Add optional support for localgov_scarfolk theme block placement out …

### PR DESCRIPTION
**Pull request by finnlewis**
…of the box.

### CODE REVIEW

File: config/optional/block.block.localgov_directories_facets.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ dependencies:
...
   theme:
-    - localgov_base
-id: localgov_directories_facets
-theme: localgov_base
+    - localgov_scarfolk
+id: localgov_directories_facets_scarfolk
+theme: localgov_scarfolk
 region: sidebar_first
 weight: 5
 provider: null
```
Comment (stephen-cox) [inline config/optional/block.block.localgov_directories_facets.yml]: This file should be renamed to  `config/optional/block.block.localgov_directories_facets_scarfolk.yml` to match the block ID.

**Review by stephen-cox — CHANGES_REQUESTED** [review]
While installing this with the profile branch that enables the Scarfolk theme and the Demo content module I see the following error message repeated twice. 

```
[error]  Block localgov_directories_channel_search_block is missing.  Cannot update its visibility settings.
```

Looks like the `CHANNEL_SEARCH_BLOCK` constant needs to be updated as well.

https://github.com/localgovdrupal/localgov_directories/blob/4ceaf0b1f73dce5e536c400cc6b7096c1be31ae4/src/Constants.php#L14

File: config/optional/block.block.localgov_directories_facets.yml (hunk 1 above)
Comment (finnlewis) [reply config/optional/block.block.localgov_directories_facets.yml]: Thanks @stephen-cox - I'd missed a commit and push. Please see now, added two new files for the localgov_scarfolk theme config.

**Comment by finnlewis** [comment]
Thanks @stephen-cox I'd missed a commit, should be good now.

**Comment by finnlewis** [comment]
Actually - what about the `CHANNEL_SEARCH_BLOCK` ? It seems to work without changing that now, as I've not changed the id of the original block.

**Comment by finnlewis** [comment]
Looking at the code, I think the block id is unchanged, as 
`localgov_directories_channel_search_block` 
even if the config id changes.

Both configs reference the same actual block plugin: 

https://github.com/localgovdrupal/localgov_directories/blob/feature/158-scarfolk/config/optional/block.block.localgov_directories_channel_search_block.yml#L14

https://github.com/localgovdrupal/localgov_directories/blob/feature/158-scarfolk/config/optional/block.block.localgov_directories_channel_search_block_scarfolk.yml#L14

**Comment by finnlewis** [comment]
Keen to merge this unless there are any objections.

**Comment by finnlewis** [comment]
@ekes any thoughts on the above?

**Comment by ekes** [comment]
> @ekes any thoughts on the above?

I can have a review of all that conditional config logic, and block placement, but @Adnan-cds probably already remembers it, so might be better placed for a quick answer.

**Review by stephen-cox — APPROVED** [review]
Looking good now

**Comment by finnlewis** [comment]
Thanks @stephen-cox ! This is why two pairs of eyes are good!

**Pull request merged**